Modules/PrinterManager/Windows/WindowsPrinterManager.cs

```diff
@@ -131,8 +132,38 @@ public override void Configure(Printer printer, bool verbose = false)
 
             if (string.IsNullOrEmpty(printer.ConfigFile)) return;
 
-            Log.Entry(LogName, "Configuring " + printer.Name);
-            PrintUI($"/Sr /n \"{printer.Name}\" /a \"{printer.ConfigFile}\" m f g p", verbose);
+            if (File.Exists(printer.ConfigFile))
+            {
+                Log.Entry(LogName, "Configuring " + printer.Name);
+                PrintUI($"/Sr /n \"{printer.Name}\" /a \"{printer.ConfigFile}\" m f g p", verbose);
+                PrintUI($"/Sr /n \"{printer.Name}\" /a \"{printer.ConfigFile}\" m f u p", verbose);
+            }
+            else {
+                // We assume the file may have been passed with parameters, eg 'C:\ConfigFile.dat m f u p', in this case, try to see if a valid file can be found.
+                try
+                {
+                    int endOfPath = printer.ConfigFile.ToLower().LastIndexOf(".dat") + 4;
+                    string varibles = printer.ConfigFile.Substring(endOfPath, printer.ConfigFile.Length - endOfPath).Trim();
```

Comment:
Just a typo in `varibles` - ought to be `variables`.

Reply:
Fixed